SuitLog/SuitLog.cs

```diff
@@ -240,11 +240,106 @@ private bool CanSuitLogRemainOpen()
                    _toolModeSwapper._firstPersonManipulator._focusedRepairReceiver == null;
         }
 
+        public string suitLogName;
         private void SetupPrompts()
         {
-            // TODO: Translations
-            _openPrompt = new ScreenPrompt(Input.PromptCommands(Input.Action.OpenSuitLog), "Open Suit Log");
-            _closePrompt = new ScreenPrompt(Input.PromptCommands(Input.Action.CloseSuitLog), "Close Suit Log");
+            // Establish these two vars
+            string openPromptString;
+            string closePromptString;
+
+            // Define them
+            switch (PlayerData.GetSavedLanguage())
+            {
+                case TextTranslation.Language.FRENCH:
+                    suitLogName = "Journal de combinaison";
+                    openPromptString = "Ouvrir le journal de combinaison";
+                    closePromptString = "Fermer le journal de combinaison";
+                    break;
+                case TextTranslation.Language.GERMAN:
+                    suitLogName = "Anzugslog";
+                    openPromptString = "Anzugslog öffnen";
+                    closePromptString = "Anzugslog schließen";
+                    break;
+                case TextTranslation.Language.ITALIAN:
+                    suitLogName = "Registro della tuta";
+                    openPromptString = "Apri il registro della tuta";
+                    closePromptString = "Chiudi il registro della tuta";
+                    break;
+                case TextTranslation.Language.JAPANESE:
+                    suitLogName = "宇宙服の航行記録";
+                    openPromptString = "宇宙服の航行記録を開く";
+                    closePromptString = "宇宙服の航行記録を閉じる";
+                    break;
+                case TextTranslation.Language.KOREAN:
+                    suitLogName = "우주복 일지";
+                    openPromptString = "우주복 일지 열기";
+                    closePromptString = "우주복 일지 닫기";
+                    break;
+                case TextTranslation.Language.POLISH:
+                    suitLogName = "Dziennik skafander";
+                    openPromptString = "Otwórz dziennik skafander";
+                    closePromptString = "Zamknij dziennik skafander";
+                    break;
+                case TextTranslation.Language.PORTUGUESE_BR:
+                    suitLogName = "Diário de Traje";
+                    openPromptString = "Abrir o Diário de Traje";
+                    closePromptString = "Fechar o Diário de Traje";
+                    break;
+                case TextTranslation.Language.RUSSIAN:
+                    suitLogName = "Скафа́ндржурнал";
+                    openPromptString = "Открыть скафа́ндржурнал";
+                    closePromptString = "Закрыть скафа́ндржурнал";
```

Comment:
Did you pick translations from the game, or used translator?
```suggestion
                    suitLogName = "Бортжурнал";
                    openPromptString = "Открыть бортжурнал";
                    closePromptString = "Закрыть бортжурнал";
```